diesel_cli/src/main.rs

```diff
@@ -352,10 +352,14 @@ fn run_infer_schema(matches: &ArgMatches) -> Result<(), Box<Error>> {
         })
         .collect();
 
-    if matches.is_present("whitelist") {
-        config.filter = Filtering::Whitelist(filter)
-    } else if matches.is_present("blacklist") {
-        config.filter = Filtering::Blacklist(filter)
+    if matches.is_present("whitelist") || matches.is_present("blacklist") {
+        eprintln!("Deprecation warning: whitelist/blacklist will be removed in a future version, use only-tables/except-tables instead.");
```

Comment:
Can we give a specific error message for each one? I'd prefer something like this:

```
if matches.is_present("whitelist") {
    eprintln!("The `whitelist` option has been deprecated and renamed to `only-tables`. ");
}

if matches.is_present("blacklist") {
    eprintln!("The `blacklist` option has been deprecated and renamed to `only-tables`. ");
}
```